dev/airflow-github

```diff
@@ -266,7 +266,9 @@ def compare(target_version, github_token, previous_version=None, show_uncherrypi
             url="URL",
         )
     )
-
+    milestone_issues = sorted(
+        milestone_issues, key=lambda x: x.closed_at if x.closed_at else x.created_at, reverse=True
```

Comment:
I didn't understand the need of `reverse` here:
> (older commits first)

Reply:
Just so I can cherrypick from the bottom of the list upwards. The last item would be the oldest

Reply:
The sorting is to get an order based on date then reverse it so that oldest commit is listed last

Reply:
Okay, I see!